src/targets/node/nvmResolver.ts

```diff
@@ -87,6 +89,17 @@ export class NvmResolver implements INvmResolver {
       }
     }
 
+    if (!directory) {
+      const fnmDir =
+        this.platform === 'win32'
+          ? path.join(this.env['APPDATA'] || '', 'fnm')
```

Comment:
It looks like `FNM_DIR` gets set on Windows as well, so we should use that here if set.

Reply:
thanks for the review, solved it!